### ISSUE TITLE
Cognate Judgments on complex words

### ISSUE THREAD

**Issue by LinguList** [digling/sinotibetan#38]
Just read the suggestions in [Starostin 2013](http://bibliography.lingpy.org?key=Starostin2013a), and they won't really help us completely, as he indicates that there are certain cases which are simply hard to solve. His most interesting example is "eye of the day/sun" vs. "sun" vs. "eye of somethingelse", where he argues that the transition that "eye of the sun" vs. "sun" is no clean transition in terms of lexical replacement, as the word is not completely lost, so they should be coded as cognate, while in the latter case, the "eye of somethingelse", the "eye" part is not really belonging to the main component, and a real lexical replacement happened. This is a similar argument, why we would say that in Chinese (hypothetical) "ritou" vs. "ri" vs. "yangtou" we would have "yangtou" as innovation/replacement, non-cognate with the other two.

I think it is best to go through the 200 concepts and to proceed accordingly (searching for the main bearer of the meaning we want to investigate, ignoring matching morphemes in non-main-components), and list all those cases which are problematic as github-issues, to further discuss them in the team. I can, by then, also make this a private repository, so that we can discuss quietly.

**Comment by LinguList** [comment]
A short update. Me and @laiyunfan will test a new way of making these judgments more explicit, at least for shallower subgroups. The idea is to:

1 test inter-annotator agreement on the set of Chinese dialects plus Old Chinese, which we want to add anyway (Chinese dialect data has already been annotated for partial cognacy by me, and @laiyunfan will also double-check those judgments)
2 test how to consistently **annotate** those decisions. The current idea is to set up a list of rules for checking (all words are mono-syllabic: no problem, one basic morpheme with pre- or suffixex which vary: basic morpheme wins, etc.). 

I'll initially test some way to make these cases explicit with an improved edictor annotation for cognates. Then we'll test, and ideally, we will be able to show that this is more or less consistent, but also transparent, so that colleagues can always nail us down on our decisions and criticize or improve them. I'll try to get the data and the annotation idea finalized by the end of this week. We are good in time with cognatizing Sinitic, as the partial cognates already take a way the bulk of the problems and make it easy to annotate normal cognates, but ideally we'll ship a good set of cognates with Old Chinese to the big database, so that @rgyalrong and @laurentsagart can have a look at potential deeper candidates (like "egg", etc.) with other subgroups which survived in some of the Chinese dialects.

**Comment by laurentsagart** [comment]
Sorry I can't understand this. Do you mean checking agreement on transcription in case we're using  two lists of words from the same dialect/language, transcribed by different people ? or checking agreement on cognate decisions made by two of us ?

**Comment by LinguList** [comment]
Here's what we do: Data is already annotated for partial cognacy in Chinese dialects. We decided on a new way to make explicit how we get from partial cognates (which are intransitive in parts) to binary cognates (the normal ones, which are transitive). These rules, are transparent and can even be computed (at least we hope so). But to make sure that our decisions go into the same direction, and since we are good in time, I thought it would be good if me and Yunfan test the consistency of this idea by converting from partial cognates to binary cognates (=full cognates). We'll report the results and details, once we have finished working on this. Just now, the EDICTOR has been set up to make this coding easier.
